lib/resources/dstu2/document_reference.yaml

```diff
@@ -35,7 +35,20 @@ fields:
         }
       ]
     }
-  note: The type must include a loinc coding.
+
+              <b>(OR)</b>
+
+    {
+      "coding": [
+        {
+          "system": "https://fhir.cerner.com/0b8a0111-e8e6-4c26-a91c-5069cbc6b1ca/codeSet/72",
+          "code": "3374547"
+        }
+      ]
+    }
+  note: The type must include a LOINC or a proprietary coding. Multiple LOINC codings or a single proprietary coding can be provided.
+        <br/><br/>
+        When providing proprietary code system, it should be of format 'https://fhir.cerner.com/&lt;your EHR source id&gt;/codeSet/&lt;code set&gt;' (where code set is a Millennium code set) and the code should be a Millennium code value. Example&#58; 'https://fhir.cerner.com/0b8a0111-e8e6-4c26-a91c-5069cbc6b1ca/codeSet/72'.
```

Comment:
'and the code should be a Millennium code value' no longer applies

Reply:
Removed: https://github.com/akhilbukka/fhir.cerner.com/commit/e1c33603617e33993b6e48bed7d6ac32e3b983ec#diff-eefc10b8b4a495f72e678cead5851448R50